docs/en/ingest-arch/14-ea-ls-es.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[ls-networkbridge]]
-=== {agent} to {ls} to {es}: {ls} to bridge networks
+=== {agent} to {ls} to {es}: {ls} as proxy to bridge networks
```

Comment:
```suggestion
=== {agent} to {ls} to {es}: {ls} as proxy
```

Reply:
Nit: Should there be an "a" here?
```suggestion
=== {agent} to {ls} to {es}: {ls} as a proxy
```

Reply:
This commit is intended to compare LS-as-proxy with the generic proxy architecture. To that end, should I shorten the title to emphasize LS as proxy for a broader range of applications?  

We could continue outlining the use case to bridge networks, and then possibly flesh out additional use cases if that makes sense.